### ISSUE TITLE
"failed to compile `bootimage v0.4.3`"

### ISSUE THREAD

**Issue by Armour0** [rust-osdev/bootimage#14]
I cant cargo install bootimage it throws out these errors
`error: expected one of `,` or `as`, found `::`
  --> .cargo\registry\src\github.com-1ecc6299db9ec823\bootimage-0.4.3\src\config.rs:27:17
   |
27 |     use std::{fs::File, io::Read};
   |                 ^^ expected one of `,` or `as` here

error: expected one of `;` or `as`, found `::`
  --> .cargo\registry\src\github.com-1ecc6299db9ec823\bootimage-0.4.3\src\config.rs:27:17
   |
27 |     use std::{fs::File, io::Read};
   |                 ^^ expected one of `;` or `as` here

error: expected item, found `;`
  --> .cargo\registry\src\github.com-1ecc6299db9ec823\bootimage-0.4.3\src\config.rs:27:34
   |
27 |     use std::{fs::File, io::Read};
   |                                  ^

error: aborting due to previous error(s)`

**Comment by phil-opp** [comment]
Seems like your compiler doesn't understand nested import groups. They were added [in Rust 1.25](https://blog.rust-lang.org/2018/03/29/Rust-1.25.html#whats-in-1250-stable), so maybe you're using a too old compiler?

**Comment by Zaper0** [comment]
yep updated rust nightly and it worked thanks also you should probably close solved issues
